File: solutions/python/rna-transcription/1/rna_transcription.py
def get_rna_complement(dna: str) -> str:
    complements = {'A': 'U', 'G': 'C', 'C': 'G', 'T': 'A'}
    return complements[dna]


def to_rna(dna_strand: str) -> str:
    rna_strand = []

    for nuc in dna_strand:
        rna_strand.append(get_rna_complement(nuc))

    s = ''.join(rna_strand)

    return s

File: solutions/go/rna-transcription/1/rna_transcription.go
package rnatranscription

func ToRNA(dna string) string {
	dnaToRna := map[rune]rune{
		'G': 'C',
		'C': 'G',
		'T': 'A',
		'A': 'U',
	}

	s := ""
	for _, r := range dna {
		s += string(dnaToRna[r])
	}
	return s
}
